### DISCUSSION TITLE
How can we ignore N001, N002, Nxxx sequence number during compare , other difference that are same

### DISCUSSION THREAD

**Discussion by Dr-Chapatine** [WinMerge/winmerge#1741] [Q&A]
Hi bloggers and support,

I have uploaded 2 files, that have N001, N002, Nxxx sequence numbers.
The files are identical except in snippet3, one line is deleted, and the N001, N002, Nxxx are re-sequenced.
we always want   Nxxx numbers ignore in comparisons. 
There will always be a space following the Nxxx number, so that can be use as a delimiter in a Regular expression, while I'm not sure how to write a RE or deploy it here.
Thanks

[seqno_snippet2.txt](https://github.com/WinMerge/winmerge/files/10872778/seqno_snippet2.txt)
[seqno_snippet3.txt](https://github.com/WinMerge/winmerge/files/10872779/seqno_snippet3.txt)

**Comment by sdottaka** [comment]
If there is a Plugins→Prediffer→IgnoreLeadingNumbers menu item, select it and the Nxxx at the beginning of the line will be ignored.

![image](https://user-images.githubusercontent.com/98126/222579546-6cffbcbb-17a3-4cae-a28d-5122486a43ff.png)

Since only the 32-bit version of WinMerge is currently installed with this plug-in, if the above menu item is not present, download IgnoreLeadingNumbers.sct by clicking the download icon at the following URL, and then add it to C:\Program Files Files\WinMerge\MergePlugins folder.

https://github.com/WinMerge/winmerge/blob/master/Plugins/dlls/IgnoreLeadingLineNumbers.sct

![image](https://user-images.githubusercontent.com/98126/222582525-8e0809c8-cdce-4359-a1f4-f84a0542f398.png)


After that, start WinMerge and select the Plugins→Prediffer→Ignore Leading Numbers menu item.

**Reply by Dr-Chapatine** [reply]
Problem solved.  Thank you very much for the detailed and speedy reply :)
